### DISCUSSION TITLE
Handling technical strings/error messages

### DISCUSSION THREAD

**Discussion by ann0see** [schlagmichdoch/PairDrop#152] [General]
* Related to translation feature #35 and strings like "selected peer left" (https://hosted.weblate.org/translate/pairdrop/pairdrop-spa/en/?q=has:comment).

* I believe some strings in English should be updated to sound less technical or have a more concise meaning to have a better user experience. In a software project I maintain, we have a style guide: https://jamulus.io/contribute/Style-and-Tone to have a somewhat consistent tone; I think that's overkill for PairDrop, but at least there should be some guidelines people follow. Especially for error messages.

* Moreover, it would be good to have an agreed way to comment on/add suggestions to the English source. I've marked some on Weblate, and think that's the way to go in general. However, as usually developers tend to hang out more on GitHub, discussions could also happen here, or in Weblate PRs. It might be easier for developers to make a ToDo list in Issues and list "to be changed strings" there. What do you think?

**Comment by UnifeGi** [comment]
There's also [another guideline](https://m2.material.io/design/communication/writing.html) from Material Design if that's useful.
